webapp/src/marked.tsx

```diff
@@ -284,6 +285,19 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
         }
     }
 
+    private renderVideo(content: HTMLElement){
+        pxt.Util.toArray(content.querySelectorAll('Video.ams-embed'))
+            .forEach((inlineVideo: HTMLElement)=> {
+                let player = MediaPlayer().create()
+                if (inlineVideo) {
```

Comment:
inlineVideo should always exist so I don't think there's a reason to have this if / it won't ever be false

Reply:
yes, that makes sense. thanks for feedback!